tests/datatable/test_datatable.py

```diff
@@ -0,0 +1,155 @@
+import textwrap
+from typing import List
+
+from src.pytest_bdd.gherkin_parser import DataTable
+from src.pytest_bdd.utils import collect_dumped_objects
+
+DATA_TABLE_FEATURE = """\
```

Comment:
These constants can just be inlined in the test function, as we don't reuse them, right?